app/src/main/java/com/swent/assos/ui/screens/calendar/Calendar.kt

```diff
@@ -0,0 +1,331 @@
+package com.swent.assos.ui.screens.calendar
+
+import androidx.compose.foundation.background
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.swent.assos.ui.theme.AssosTheme
+import androidx.compose.ui.Modifier
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ChevronLeft
+import androidx.compose.material.icons.filled.ChevronRight
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.Layout
+import androidx.compose.ui.layout.ParentDataModifier
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.temporal.ChronoUnit
+import kotlin.math.roundToInt
+
+@Preview(showSystemUi = true)
+@Composable
+fun CalendarAppPreview() {
+  AssosTheme {
+    Calendar(
+      events = sampleEvents,
+      modifier = Modifier.padding(16.dp)
+    )
+  }
+}
```

Comment:
The `CalendarAppPreview` is not necessary, you can call the `Calendar` composable in the `HomeNavigation.kt`. The parameters are not necessary, you can use them directly in the composable